war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2746,6 +2858,152 @@ function loadScript(href,callback) {
     head.insertBefore( script, head.firstChild );
 }
 
+/**
+ * Loads a dynamically created invisible IFRAME.
+ */
+function createIframe(src,callback) {
+    var iframe = document.createElement("iframe");
+    iframe.src = src;
+    iframe.style.display = "none";
+
+    var done = false;
+    iframe.onload = iframe.onreadystatechange = function() {
+        if ( !done && (!this.readyState ||
+                this.readyState === "loaded" || this.readyState === "complete") ) {
+            done = true;
+            callback();
+        }
+    };
+
+    document.body.appendChild(iframe);
+    return iframe;
+}
+
+var downloadService = {
```

Comment:
support for uplaoding update site metadata from the browser was removed a while ago (2.204.1)
https://github.com/jenkinsci/jenkins/pull/3970/files#diff-da911448bbb85d374f604da9c1e00136

so there is either a bad merge / merge resolution or this PR is out of date.